### PR TITLE
fix issue 2634 Quotes in tables' comment field break mypostscript

### PR DESCRIPTION
**Pull request by bybai**
fix issue #2634 Quotes in tables' comment field break mypostscript

### CODE REVIEW

File: xCAT-server/lib/perl/xCAT/Postage.pm [hunk 1]
```diff
@@ -1582,7 +1582,10 @@ sub dump_all_attribs_in_tabs
                 } else {
                     $values .= "$attrib=$val||";
                     if ($attrib =~ /^disable$/) {
-                        $values .= "comments=$t";
+                        # Updated on 2017-03-22 for issue 2634 Quotes in tables' comment field break mypostscript
+                        # The original line is : $values .= "comments=$t";
+                        # In order to fix issue 2634, change this line to : $values .= "comments=";
```
Comment (immarvin) [inline xCAT-server/lib/perl/xCAT/Postage.pm]: should better be ``To keep the free-style of comments and avoid the issues caused by special characters in the comments, the value of comments will not be appending to network related environment variables in mypostscript``

**Comment by bybai** [comment]
Hi @immarvin , I commit new changes. Thanks.

**Comment by immarvin** [comment]
agree to merge,

**Comment by kcgthb** [comment]
Wouldn't it be better to just escape the single quotes in the comment field rather than dropping the comments altogether?

What happens if another field (other than "comments") contains a single quote?

**Pull request merged**